src/bundle/Controller/SearchController.php

```diff
@@ -128,6 +128,7 @@ public function searchAction(Request $request): Response
                 $contentTypes = $data->getContentTypes();
                 $lastModified = $data->getLastModified();
                 $created = $data->getCreated();
+                $filtersCollapsed = empty($contentTypes) && null == $section && empty($lastModified) && empty($created);
```

Comment:
I think that logic (checking if `$data` carries any filtering should be a responsibility of `SearchData` object)
So something like `$data->isFiltered()`. 
WDYT @mikadamczyk?